server_installation/topics/operator/extensions.adoc

```diff
@@ -0,0 +1,39 @@
+
+[[_operator-extensions]]
+=== Installing extensions and themes
+
+You can use the operator to install extensions and themes that you need for your company or organization. The extension or theme can be anything that {project_name} can consume. For example, you can add a metrics extension. You add the extension or theme to the Keycloak custom resource.
```

Comment:
Keycloak custom resource -> I have a question @hmlnarik @slaskawi. In this project there are many places where is mentioned Keycloak. Shall we parametrize also Keycloak custom resource => {project name} custom resource? Then the product will look like this: RHSSO custom resource. I know that api or kind will remain. But in case of CR files. What do you think?

Reply:
Definitely not! It's the same pattern as we use in RHSSO for Keycloak classes and jars. They all are named ".*keycloak.*". Otherwise it will be a gigantic maintenance headache.

Reply:
I am asking also, because when you start it from template, there is rh-sso label and the operators shall replace templates in Openshift as far as I know.